ncm-ssh/src/main/perl/ssh.pm

```diff
@@ -74,16 +83,21 @@ sub handle_config_file
             my $ref = ref($val);
             if($ref) {
                 if($ref eq 'ARRAY') {
+                    if ($option_set eq 'multiline_options') {
+                        $fh->remove_lines(qr{(?i)^\W*$option(?:\s+\S+)+}, qr{^#});
```

Comment:
What do you mean by remove content that you add below?

The intention (and it appeared to work) was to remove all existing instances of an option in the file, and then add them back in. This matches the behaviour of regular options, if you set an option then you expect an existing one (or multiple in this case) of that name to be replaced.

I looked at CAF::FileWriter and couldn't figure anything that would fit multiline options, like add_or_replace does for the regular ones, other than doing this.